dto/StatisticsDTO.java

```diff
@@ -17,6 +17,10 @@ public StatisticsDTO(String sum, String avg, String max, String min, Long count)
         this.count = count;
     }
 
+    public StatisticsDTO() {
```

Comment:
????? why empty, is it neeeded?

Reply:
Because when spring makes the serialization/deserialization JSON/Object it needs the empty constructor. If it is not present fails.